src/routes/PetPage/PetPage.js

```diff
@@ -0,0 +1,152 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Section } from '../../components/Utils/Utils';
+import PetsService from '../../services/petsService';
+import { Link } from 'react-router-dom';
+
+export default function PetPage () {
+  const params = useParams();
+  const [pet, setPet] = useState(null);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    if (!pet) {
+      PetsService.getPet(params.petID)
+        .then(pet => setPet(pet))
+        .catch(e => {
+          setError(e.error);
+        });
+    }
+  });
+
+  const renderFavoriteIcon = () => {
+    // TODO
+    if (true) {
+      return <span className='text-danger'><FontAwesomeIcon icon='heart' /></span>;
+    }
+    return <span className='text-danger'><FontAwesomeIcon icon={['far', 'heart']} /></span>;
+  }
+
+  const renderPetBio = () => {
```

Comment:
I think we should maybe have a place to put the animal type: dog, cat, other. Thoughts?

Reply:
Definitely! I totally missed it lol